futures/tests_disabled/all.rs

```diff
@@ -376,6 +376,106 @@ fn select2() {
     }
 }
 
+#[test]
```

Comment:
`tests_disabled` contains tests that we haven't ported to 0.3 yet. Our goal is to look at each test and migrate it to 0.3 (or discard if it doesn't apply anymore). (BTW PRs are appreciated 😁)


Adding new tests into this folder isn't a good idea 🙂

Reply:
Yep, in the process of attempting to move them out.